plone/app/iterate/browser/configure.zcml

```diff
@@ -45,7 +45,7 @@
        class=".diff.DiffView"
        name="iterate_diff"
        template="diff.pt"
-       permission="zope2.View"
+       permission="cmf.ModifyPortalContent"
        />
```

Comment:
@pbauer I found this old PR while testing https://github.com/plone/Products.CMFPlone/issues/1452. The style updates look good to me. The permission updates also look desirable but I think they will need a bit more work to ensure the rolemap is set up correctly; could you separate out that change?